README.md

```diff
@@ -9,20 +9,23 @@ The Collective Experience (CE) Platform facilitates the creation and operation o
 5. Start the server using `meteor`.
 
 Hacky way in case meteor installation doesnt work (for Windows):
```

Comment:
@mtruong68 forgot something: can you add a mention to the start script too? We're using `npm start` to do some of these commands.